flask/app/templates/play.html

```diff
@@ -1,6 +1,7 @@
 {% extends 'base.html' %}
 {% set active_page = "play" %}
 {% block title %} Game {% endblock %}
+{% block head_css %} <link rel="stylesheet" href="{{ url_for('static', filename='styles/index.css') }}" /> {% endblock %}
```

Comment:
please rename to `play.css`. `index.css` implies that it is meant for the main page